erpnext/stock/doctype/shipment/shipment.js

```diff
@@ -381,7 +376,18 @@ frappe.ui.form.on('Shipment', {
 				frappe.throw(__("Pickup Time cannot be in the past"));
 			}
 		}
-		frm.trigger('set_pickup_to_time');
+	},
+	pickup_to: function(frm) {
```

Comment:
I think it would be better to just remove `pickup_to`, `pickup_from` and `get_pickup_time`. 

There's a server-side validation for this: https://github.com/frappe/erpnext/blob/39b1cd827a5e3ba04c29189426e5f85f7fe77daf/erpnext/stock/doctype/shipment/shipment.py#L36-L38